package.json

```diff
@@ -20,21 +20,29 @@
     "type": "git",
     "url": "git://github.com/brianloveswords/buffer-crc32.git"
   },
-  "main": "index.js",
   "scripts": {
-    "test": "tap tests/*.test.js --reporter classic"
+    "test": "tap tests/*.test.js --reporter classic",
+    "build": "unbuild",
+    "prepublishOnly": "npm run build"
   },
-  "type": "commonjs",
```

Comment:
I've removed explicit type, because it can actually be easier ESM or CommonJS, but we can't default it to ESM, because otherwise tests break, as older TAP does not support CJS format, and older Node versions don't support ESM